edb/schema/modules.py

```diff
@@ -29,7 +29,9 @@
 
 class Module(so.UnqualifiedObject, s_anno.AnnotationSubject,
              qlkind=qltypes.SchemaObjectClass.MODULE):
-    pass
+
+    builtin = so.SchemaField(
```

Comment:
Setting of this field should be protected by `stdmode`

Reply:
`allow_ddl_set=False` for this field.